src/BaselineUtils.cc

```diff
@@ -1037,8 +1129,16 @@ symmetrizeFootprint(
         if (dxlo <= dxhi) {
             LOGL_DEBUG(_log, "Adding span fwd %i, [%i, %i],  back %i, [%i, %i]",
                        fy, cx+dxlo, cx+dxhi, by, cx-dxhi, cx-dxlo);
-            sfoot->addSpan(fy, cx + dxlo, cx + dxhi);
-            sfoot->addSpan(by, cx - dxhi, cx - dxlo);
+            if (cx + dxlo <= cx + dxhi) {
+                tmpSpans.push_back(geom::Span(fy, cx + dxlo, cx + dxhi));
+            } else {
+                tmpSpans.push_back(geom::Span(fy, cx + dxhi, cx + dxlo));
+            }
```

Comment:
I must be missing something, why is this extra if..else block needed? Line 1129 checks if ``dxlo<=dxhi``, so why would ``cx+dxlo<=cx+dxhi`` give a different result?